mumbleBot.py

```diff
@@ -398,6 +398,7 @@ def message_received(self, text):
 			                     'url': url,
 			                     'user': user}
                         var.playlist.append(music)
+                        self.send_msg('Now playing ' + radiobrowser.getstationname_byid(parameter), text)
```

Comment:
Now playing should be done late by the bot.
https://github.com/azlux/botamusique/blob/master/mumbleBot.py#L518

Reply:
I´m thinking of replacing the bot´s message against a table with information about the radio station from radio-browser, which is more accurate and contains more information.

Any idea how to disable the bot message when playing a rb station?

_Bot message_
```
[22:43:47] (Kanal) botamusique: Now playing http://regiocast.hoerradar.de/nora-103-mp3-mq?sABC=5q3p46ps%230%23o000r827s931po57p4q48q3377r73615%23ubzrcntr&amsparams=playerid:homepage;skey:1564231375
URL : http://regiocast.hoerradar.de/nora-103-mp3-mq?sABC=5q3p46ps%230%23o000r827s931po57p4q48q3377r73615%23ubzrcntr&amsparams=playerid:homepage;skey:1564231375
```
_RB message_

| ID | Station Name | Genre | Codec/Bitrate | Country | Homepage |
| -- | -- | -- | -- | -- | -- |
| 96748 | NORA Oldies | 60s,70s | MP3/128 | Germany | http://www.norawebstreams.de/ |

Reply:
Shall I remove the message? Otherwise: No more commits from me concerning this pr.